### ISSUE TITLE
loadingWidgetBuilder adds a lot of space under content

### ISSUE THREAD

**Issue by Seamate** [shah-xad/flutter_tex#129]
Using the loadingWidgetBuilder to show a text or spinner while data loads puts a lot of space under the content when loading completes.

How can we remove this unwanted space under? 

Here is the code
`Container(
          margin: EdgeInsets.only(top: 5),
          padding: EdgeInsets.all(10),
          width: double.infinity,
          child: TeXView(
            
            style: TeXViewStyle(fontStyle: TeXViewFontStyle(fontSize: 25)),
            renderingEngine: const TeXViewRenderingEngine.mathjax(),
            child: TeXViewDocument(questionText),
          ),
        ),`

Please see the image below.

![image](https://user-images.githubusercontent.com/35474522/172781151-3e680b3f-f04e-4780-8448-b367519f8a72.png)

**Comment by bijoya-banik** [comment]
Have you solved this issue?

**Comment by Seamate** [comment]
@bijoya-banik Yes, I solved the issue with a different package. I had to discontinue using flutter_tex as obviously, the package is no longer supported.
Thanks for your concern

**Comment by vanman98** [comment]
What package did you use instead?

**Comment by todaysprint** [comment]
@Seamate  which package please let us know

**Comment by shahabas-cp-14** [comment]
> @bijoya-banik Yes, I solved the issue with a different package. I had to discontinue using flutter_tex as obviously, the package is no longer supported. Thanks for your concern

which package ?
